File: ruby/fizzbuzz.rb
=begin
	FizzBuzz

	Run with:
	$ ruby fizzbuzz.rb
=end

(1..100).each{|i|
	x = ''
	x += 'Fizz' if i%3==0
	x += 'Buzz' if i%5==0
	puts(x.empty? ? i : x);
}

File: php/fizzbuzz.php
#!/usr/local/bin/php
<?php

/*
	FizzBuzz

	Run with:
	$ php fizzbuzz.php
 */

$i = 1;
while ($i <= 100):
	if( $i % 15 == 0 ) echo "FizzBuzz\n";
	elseif( $i % 3 == 0 ) echo "Fizz\n";
	elseif( $i % 5 == 0 ) echo "Buzz\n";
	else echo $i . "\n";
	$i++;
endwhile;
